src/Entity.php

```diff
@@ -559,6 +559,17 @@ public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $
         return true;
     }
 
+    public function showForm($ID, array $options = [])
+    {
+        if ((int)$ID === 0) {
+            // Root entity: can edit but cannot delete
+            $options['canedit'] = true;
```

Comment:
Is this normal ? An user could see the root entity without having the rights to edit it.

Reply:
This piece of code was present in the previous file.